### PR TITLE
Reformat all files with new prettier config

### PR DESCRIPTION
**Pull request by danielduong8**
I have deleted a single line of code somewhere without knowing it, your task is to help me find it. Good luck.

### CODE REVIEW

File: client/src/components/CreatePost.test.tsx [hunk 1]
```diff
@@ -1,14 +1,13 @@
-import React from "react";
-import CreatePost from "./CreatePost";
-import { unmountComponentAtNode } from "react-dom";
-import { act } from "react-dom/test-utils";
-import { render, screen, fireEvent, cleanup } from "@testing-library/react";
-import newImage from "../assets/background.jpg";
```
Comment (danielduong8) [inline client/src/components/CreatePost.test.tsx:6]: This was an unused import, deleting this was intentional.

**Comment by danielduong8** [comment]
Im kidding btw about the PR message

**Review by andy9775 — APPROVED** [review]
Can you add the following rule: https://prettier.io/docs/en/options.html#bracket-spacing. Either true or false is ok, I'll leave it up to you to decide. After you do so, re-run the formatter. 

I also suggest everyone turn on [auto format on save](https://linuxpip.org/vscode-format-on-save/).

Otherwise LGTM!

**Review by plausibly — APPROVED** [review]
This better not cause merge conflicts

**Comment by andy9775** [comment]
@plausibly should have been the first to take up the task. Don't have to deal with conflicts and get to add to that contribution graph.

**Comment by danielduong8** [comment]
> @plausibly should have been the first to take up the task. Don't have to deal with conflicts and get to add to that contribution graph.

How come I don't see anyone talking about andy's contribution graph 👀

**Pull request merged**